translator/translate/otel/pipeline/host/translators.go

```diff
@@ -0,0 +1,47 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+package host
+
+import (
+	"fmt"
+
+	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/confmap"
+
+	"github.com/aws/amazon-cloudwatch-agent/receiver/adapter"
+	"github.com/aws/amazon-cloudwatch-agent/translator/translate/otel/common"
+	"github.com/aws/amazon-cloudwatch-agent/translator/translate/otel/pipeline"
+	adaptertranslator "github.com/aws/amazon-cloudwatch-agent/translator/translate/otel/receiver/adapter"
+)
+
+func NewTranslators(conf *confmap.Conf, os string) (pipeline.TranslatorMap, error) {
+	adapterReceivers, err := adaptertranslator.FindReceiversInConfig(conf, os)
+	if err != nil {
+		return nil, fmt.Errorf("unable to find receivers in config: %w", err)
+	}
+
+	translators := common.NewTranslatorMap[*common.ComponentTranslators]()
+	// split out delta receiver types
+	deltaReceivers := common.NewTranslatorMap[component.Config]()
```

Comment:
whats the difference between delta receivers and host receivers?

Reply:
Delta metrics require the delta processor. They are a subset of the telegraf metrics. 

translator.ID().Type() == adapter.Type(common.DiskIOKey) || translator.ID().Type() == adapter.Type(common.NetKey)

Reply:
okay so basically they use telegraf?